_drafts/2018-04-18-q1-results.md

```diff
@@ -0,0 +1,70 @@
+
+## Preview component and AD Sync
+
+In Q1 an important decision was made concerning the Enterprise only features of sensenet, therefore we will publish each and every feature as part of the Community Edition, including AD Sync and Preview Component as well.
+
+For that reason the [Preview component](https://www.nuget.org/packages/SenseNet.Preview/) and [AD Sync](https://www.nuget.org/packages/SenseNet.SyncAD2Portal) NuGet packages are available for any sensenet 7 solution even without having Enterprise licences.
+
+## First SN7 patch
+![sn7 patch](/img/posts/sn_patch_bear.jpg "sn7 patch")
+In February the first patch for sensenet 7 has arrived with several bugfixes and features.
+At the time of writing, the sensenet 7.1.1 version is already out.
+Follow releases on [GitHub](https://github.com/SenseNet/sensenet/releases) to keep up to date.
+Read our [tutorial](https://community.sensenet.com/blog/2018/02/14/install-patch) about installing patches in sensenet.
+
+
+## Scoped packages
+
+The whole idea behind scoped packages was to divide overgrown client side packages into smaller ones and publish these new, refactored packages under @sensenet scope.
+Simplifying API and get rid of external dependencies like RxJs library were also among our goals when we started this project a few months ago.
+
+Take a look at our new packages and their dependencies:
+![Scoped packages](/img/posts/scoped-packages-dependencies.png "scoped packages dependencies")
+[Learn more](/blog/2018/02/21/scoped-packages) about what we achieved in Q1 with npm packages.
+
+
+## Release sn7.1
+Back in March, [sensenet 7.1](https://github.com/SenseNet/sensenet/releases/tag/v7.1.0) was published packed with many under the hood changes. Major changes were made in indexing and querying generalization. All of these modifications are needed to prepare Lucene search engine to work as a service instead as a local index. In this first phase no changes have been made in architectural level, however you may need to rebuild or update some libraries before executing the patch due to some API changes.
+Please read our [post](https://community.sensenet.com/blog/2018/03/05/complex-patch) to learn more about the changes and the installation process.
+
+## Testing
+We have a huge amount of tests in sensenet, that's why their runtime and effectiveness are really important. Before SN7, we used more than 1400 tests and most of them were integration tests which required a fully installed sensenet and lot of time to run. We decided to [start over](/blog/2017/10/27/in-memory-testing) and use unit tests to increase effectiveness of testing and development.
+The new tests are intended to test small pieces of the source code without depending on other parts, therefore we can quickly locate errors without reviewing the whole code. Another important point is that the runtime of these tests is now much shorter so we are able to run more tests in less time.
```

Comment:
A few words of a conclusion, a closure would be nice, for example a hint that there is another post coming about the next quarter.